libs/common/src/types/mongo-db-models/deals.ts

```diff
@@ -8,9 +8,11 @@ type BssEwcsDeal = AnyObject & {
   };
 };
 
-type GefDeal = AnyObject & {
+export type GefDeal = AnyObject & {
   dealType: typeof DEAL_TYPE.GEF;
-  ukefDealId: string;
+  ukefDealId: string | null;
```

Comment:
`ukefDealId?` will not work right?

Reply:
Nope, this is set in `portal-api\src\v1\gef\models\application.js` as `null` rather than `undefined`